docs/source/high-level-features.rst

```diff
@@ -28,7 +28,7 @@ Code generation
 
 Palm includes code generation functionality, powered by jinja. Code generation
 allows you to automate repetitive boiler plating tasks, and keep your codebase
-consistent. Code generation is driven by the ``ctx.obj.generate()`` function.
+consistent. Code generation is driven by the ``environment.generate()`` function.
```

Comment:
and here

Reply:
Same as above that was actually an Environment `generate` not a Containerize `generate`?